src/forever.js

```diff
@@ -1,28 +1,52 @@
 'use strict';
 var src = require('./program');
+var graphite = require('graphite');
+var utils = require('./lib/utility.js');
+var graphiteClient;
 
 module.exports = function(ipp, dbUrl, commander) {
   console.log("FOREVER: calling ENTER.....")
+
+  if(commander.graphite){
+    graphiteClient = graphite.createClient('plaintext://54.91.39.21:2003/');
```

Comment:
The graphite address should be passed in as an environment variable or command line argument